code/modules/mob/living/simple_animal/abnormality/aleph/mountain.dm

```diff
@@ -74,10 +76,48 @@
 	return ..()
 
 /mob/living/simple_animal/hostile/abnormality/mountain/Move()
+	var/turf/floor = get_turf(src)
+	for(var/thing in floor)
+		if(istype(thing, /obj/effect/decal/cleanable/blood))
+			var/obj/effect/decal/cleanable/blood/blood_decal = thing
+			adjustBruteLoss(-clamp((maxHealth*0.005)*(blood_decal.bloodiness*0.01), 0, maxHealth)) // Heal up to 0.5% heath from bloody ground.
+			QDEL_NULL(blood_decal)
+			belly += 0.01
+			StageChange(TRUE)
+			continue
+		if(istype(thing, /obj/item/organ))
+			var/obj/item/organ/floor_organ = thing
+			adjustBruteLoss(-clamp((maxHealth*0.01), 0, maxHealth)) // Heal 1% health from Organs
+			QDEL_NULL(floor_organ)
+			belly += 0.05
+			StageChange(TRUE)
+			continue
+		if(istype(thing, /obj/item/bodypart))
+			var/obj/item/bodypart/floor_part = thing
+			adjustBruteLoss(-clamp((maxHealth*0.01), 0, maxHealth)) // Heal 1% health from Body Parts
+			QDEL_NULL(floor_part)
+			belly += 0.05 // Progresses toward getting bigger
+			StageChange(TRUE)
+			continue
 	if(finishing)
 		return FALSE
+	new /obj/effect/decal/cleanable/old_flesh(floor)
```

Comment:
Add a check so there is only one such effect on the floor.

Reply:
Doesnt need a check, really. Just needs a variable changed in the decal